### ISSUE TITLE
REL-3 — Docs consolidation for v1 and retirement of the separate `docs` branch

### ISSUE THREAD

**Issue by aacostadiaz** [ACEsuit/mace#1605]
**Depends on:** REL-1 (#1603) · **Blocks:** —

**Prerequisite:** the Phase 0 characterization suite is committed — goldens, fixtures and the single-source tolerance table under `tests/golden/`, characterization tests under `tests/unit/`. References of the form `P0-N` name that work.

**Context:** User documentation for MACE does not live with the code. It lives on a separate `docs` branch (`upstream/docs`, tip `2413caaa`) — a **full stale fork of the repository**, **1612 commits behind `develop`** at `3b43d63f` — that carries a `docs/` folder plus a `.readthedocs.yml` on top of that old snapshot. `develop` itself has **no `docs/` directory and no `.readthedocs.yml`**. This split is why the docs drift: doc changes cannot ride with the code PR that changes the behaviour, tutorials are never executed against the code, and the branch accumulates doc-only work that never reaches the main tree — **188** commits exist on `docs` and not on `develop`, **172** of which touch `docs/`. This ticket ends the split for v1: bring `docs/` **into the main tree** so docs are versioned with the code and doc PRs ride with code PRs, point readthedocs at the in-tree `docs/`, make the tutorials execute in CI, rewrite the pages for the v1 stack, and delete the `docs` branch. This is the last documentation ticket before the release: it lands after the migration guide and DROP list are published (REL-1 (#1603)) and its tutorials are executed against the v1.0.0 build.

**Interface & constraints:**

- **Consolidate, do not merge the fork.** The `docs` branch is a whole-repo snapshot 1612 commits behind; merging it would resurrect stale code. Instead **cherry-pick the `docs/`-folder changes only** — port the 172 doc-touching commits as content into the in-tree `docs/`, then delete the branch. The `.readthedocs.yml` and `docs/requirements.txt` move in with the tree and are repointed at the in-tree `docs/`.
- **Every user-promised page must exist and be correct for v1.** The branch's current page set is the starting inventory: `docs/index.rst`, `docs/quick-start/quick-start.rst`, `docs/guide/{intro,installation,troubleshooting,training,evaluation,descriptors,hessian,ase,foundation_models,finetuning,lora_finetuning,multihead_finetuning,multihead_training,heterogeneous_data,cuda_acceleration,lammps,lammps_mliap,openmm,polar_mace,polarizability,multipreprocessing,multigpu}.rst`, and `docs/examples/{ase_examples,foundation_examples,training_examples,tutorials}.rst`. Each maps to KEEP functionality and each must reflect the v1 stack, not legacy. Pages that describe mechanisms v1 removed are **rewritten**, not copied:

| Doc page | v1 note |
|---|---|
| `quick-start`, `guide/intro`, `guide/installation`, `guide/troubleshooting` | package install (`packages/` names decided at INF-1 (#1550)), `--engine {legacy,v1}` / `MACE_ENGINE=v1` |
| `guide/training` | v1 training entry point + config system; the LBFGS regime is a config choice, not a flag |
| `guide/evaluation`, `guide/descriptors`, `guide/hessian` | eval CLI + `get_descriptors` + analytical `get_hessian`; state which model types each supports (descriptors: energy models; hessian: energy + polar) |
| `guide/heterogeneous_data`, `guide/multihead_training` | pairs with FT-2 (#1581) docs |
| `guide/ase` | the v1 calculator (DEP-1 (#1583) / P0-5 contract), including the graph-padding knobs |
| `guide/polarizability` | dipole/polarizability observables on the v1 architecture (the MDP model's home) |
| **`guide/cuda_acceleration`** | **rewritten**: v1 dispatches backends (cueq/oeq) at build time — there is **no weight conversion** step; the five legacy `convert_*` CLIs are gone. Do not document e3nn↔backend conversion |
| `guide/openmm` | DEP-3 (#1595) |
| `guide/lammps`, `guide/lammps_mliap` | **merged and rewritten**: MLIAP is the only path (DEP-2 (#1594)), the bundle + manifest replace `-lammps.pt`, and the KOKKOS requirement for multi-layer models is stated on the page rather than discovered at runtime |
| `guide/foundation_models` | loaders + the one-shot converter story (pairs with FM-3 (#1588)); the 25-artifact roster and the ANI-CC drop |
| `guide/polar_mace` | `[electrostatics]` extra + PolarMACE on v1 (ELEC-1 (#1591)/2) |
| **new: LES / electrostatics-LES page** | `[les]` extra, BEC, latent multipoles, external field (ELEC-4 (#1635)) — undocumented today and a regression if it stays that way |
| **new: magnetic MACE page** | magmom input, magnetic-moment observable, `dE/dm`, the SCF variant (MAG-1 (#1584)) — the family has 31 tests and a CI job and **no doc page at all** on the `docs` branch |
| **new: torch-sim page** | first-class deployment path (DEP-4 (#1596)), alongside ASE/LAMMPS/OpenMM |
| `guide/multipreprocessing` | v1 preprocessing/sharding |
| `guide/multigpu` | distributed training |
| `docs/examples/*` + Tutorials 1–2 | end-to-end usage; the old Tutorial 3 (theory) is superseded by the EDU-2 (#1608) theory tutorial |

  (Fine-tuning / Multihead Replay / LoRA docs — `guide/finetuning`, `guide/lora_finetuning`, `guide/multihead_finetuning` — are owned by FT-2 (#1581), not this ticket.)
- **CI-tested tutorials (hard rule, shared with GOV-1 (#1609)).** Tutorials and examples are executed **end-to-end in CI** at merge time, so a doc page that shows a command runs that command. A PR that breaks an example must fix the example in the same PR. Prefer marimo notebooks over Jupyter for the executed tutorials. **Assign each page a tier.** The per-PR docs job runs every documented command on a subsampled fixture with a bounded step count and no dataset downloads; the full-scale example runs (MD22, ANI-1x, liquid water, NVT with a foundation model) run in `nightly.yaml` with `allow-network: true`, the same way the `foundations` job already opts in via `MACE_CI_ALLOW_NETWORK=1`. "Executed end-to-end" means the documented command runs to completion at its assigned tier — reproducing the published numbers is a nightly obligation, not a per-PR one. There is **no JAX training** anywhere; JAX appears in docs only as inference/inspection.
- **Terminology alignment.** Docs use the v1 vocabulary: the new engine is **v1** (`--engine {legacy,v1}`, `MACE_ENGINE=v1`), never "v2"; Clebsch-Gordan is spelled out, never "CG"; "stage two", never "SWA". The only legitimate mention of e3nn in v1 docs is when describing legacy behaviour or the legacy toolchain.
- **Single source of truth after this ticket.** Once merged, the `docs` branch no longer exists; readthedocs builds from the in-tree `docs/` on the main branch, so docs version with the code from then on.

**Task:**
1. Create `docs/` in the main tree (with `.readthedocs.yml` and `docs/requirements.txt`) and repoint the readthedocs project at the in-tree `docs/`.
2. Cherry-pick the 172 doc-touching commits from `upstream/docs` as content into the in-tree `docs/`; do **not** merge the stale branch fork.
3. Rewrite/author every v1 page in the surface table so it reflects the v1 stack — in particular rewrite the CUDA-acceleration page for build-time backend dispatch with no weight-conversion step, merge the two LAMMPS pages onto the MLIAP bundle, and write the three missing pages (LES, magnetic, torch-sim). Drop legacy-only pages that have no v1 equivalent, with the drop recorded.
4. Port the README to v1 (install, engine selection, quick start), pointing at the consolidated docs.
5. Wire the tutorials/examples into CI as end-to-end executed tests at merge time (marimo preferred); make the Examples set and Tutorials 1–2 run; leave the theory tutorial to EDU-2 (#1608).
6. Execute the consolidated docs/tutorials against the v1.0.0 build and fix any drift found.
7. Delete the `docs` branch once the in-tree docs build green on readthedocs.

**Out of scope:** the migration guide, DROP list, and AI migration skill file (REL-1 (#1603)); fine-tuning / multihead-replay / LoRA doc pages (FT-2 (#1581) docs); the EDU-1 (#1607) `tutorials/` JAX inspection scaffold and the EDU-2 (#1608) theory tutorial + marimo cross-check; CONTRIBUTING/governance authoring (GOV-1 (#1609), which owns the CI-tested-tutorials *rule* — this ticket is its first consumer).

**Acceptance criteria:**
- [ ] `docs/` (with `.readthedocs.yml`) exists in the main tree; readthedocs builds from it and the build is green.
- [ ] The `docs` branch is deleted; there is no remaining out-of-tree docs source.
- [ ] The 172 doc-touching commits are ported as content — spot-checked present — without merging the stale fork.
- [ ] Every page in the v1 documentation surface exists and reflects the v1 stack; the CUDA-acceleration page describes build-time dispatch with **no** weight conversion and no e3nn↔backend conversion CLIs; the two LAMMPS pages are merged onto the MLIAP bundle and state the KOKKOS requirement.
- [ ] The three previously-undocumented capabilities — LES, magnetic MACE, torch-sim — each have a page; a meta-check asserts every capability marker in `pyproject.toml` maps to at least one doc page.
- [ ] Tutorials 1–2 and the Examples set (MD22, ANI-1x, liquid water, NVT-with-FM) execute end-to-end in CI; a deliberately broken example command fails the docs CI job.
- [ ] The docs/tutorials were executed against the v1.0.0 build and any drift is fixed.
- [ ] No page uses legacy-only terminology for v1 features (no "v2", no bare "CG", no "SWA", no e3nn except when describing legacy).

**Verify:**
```bash
# In-tree docs build (readthedocs config), CPU-only
python -m sphinx -b html docs/ docs/_build/html
# Tutorials/examples executed as CI does
python -m pytest docs -k "tutorial or example"      # end-to-end doc execution job
# No stale terminology / dead conversion references leaked into v1 pages
! grep -rniE '\bv2\b|\bcg[_ ]|\bswa\b|convert_e3nn|cueq_to_e3nn' docs --include='*.md' --include='*.rst'
# Every capability marker has a page
python -m pytest tests/architecture -k docs_capability_coverage -v
```
**Review focus:** that the CUDA-acceleration, LAMMPS and foundation-model pages describe the v1 build-time dispatch, MLIAP-only export and one-shot-converter reality (no runtime weight conversion, no `convert_*` CLIs, no `-lammps.pt`), that the three new pages actually exist rather than being deferred (LES and magnetic are shipped features with zero documentation today — publishing v1 without them repeats the gap), and that the executed-tutorials CI job actually fails on a broken example (the rule is real, not decorative).

---
